pkg/kopia/command/snapshot.go

```diff
@@ -64,7 +64,7 @@ type SnapshotRestoreCommandArgs struct {
 func SnapshotRestore(cmdArgs SnapshotRestoreCommandArgs) []string {
 	args := commonArgs(cmdArgs.CommandArgs, false)
 	args = args.AppendLoggable(snapshotSubCommand, restoreSubCommand, cmdArgs.SnapID, cmdArgs.TargetPath)
-	args = args.AppendLoggableKV(ignorePermissionsError, strconv.FormatBool(cmdArgs.IgnorePermissionErrors))
+	args = args.AppendLoggableBool(ignorePermissionsError, cmdArgs.IgnorePermissionErrors)
```

Comment:
@TimonOmsk Wouldn't it be simpler if we do
```
if cmdArgs.IgnorePermissionErrors {
    args = args.AppendLoggable(ignorePermissionsError)
} else {
    args = args.AppendLoggable(noIgnorePermissionsError)
}
```
`noIgnorePermissionsError` is a constant with `--no-ignore-permission-errors` value

Reply:
It's simpler, but we'd have to introduce 2 arguments for every single argument. I think it's up to you here. If you think it's easier I would do it :)

Reply:
I like @pavannd1's proposal. It's simple and straightforward.